Action fails when used with merge queue

I see the following failures when used with merge queue:

```
 + /home/runner/go/bin/go-apidiff origin/ --compare-imports=false --print-compatible=true --repo-path=.
failed to lookup git commit hashes: could not get hash for "origin/": Revision invalid : must not end with "/"
Error: Process completed with exit code 2.
```

Example: https://github.com/operator-framework/operator-controller/actions/runs/5974226156/job/16207958228

--- a/action.yml
+++ b/action.yml
@@ -7,7 +7,7 @@ inputs:
   base-ref:
     description: 'Base reference for API compatibility comparison (default: base branch)'
     required: false
-    default: "origin/${GITHUB_BASE_REF}"
+    default: "${{ github.event.pull_request.base.sha || github.event.merge_group.base_sha }}"
   version:
     description: 'Version of go-apidiff to use'
     required: false

